install-dev/theme/views/welcome.php

```diff
@@ -39,22 +39,22 @@
  ); ?></div>
 <?php endif; ?>
 
-<h2><?php echo $this->translator->trans('Welcome to the PrestaShop %version% Installer', array('%version%' => _PS_INSTALL_VERSION_), 'Install') ?></h2>
+<h2><?php echo $this->translator->trans('Welcome to the PrestaShop %version% Installer', array('%version%' => _PS_INSTALL_VERSION_), 'Install'); ?></h2>
 <p><?php echo $this->translator->trans('Installing PrestaShop is quick and easy. In just a few moments, you will become part of a community consisting of more than 250,000 merchants. You are on the way to creating your own unique online store that you can manage easily every day.', array(), 'Install'); ?></p>
```

Comment:
Can we change the wording here?

Reply:
Sure. How would you like the wording?

Reply:
Exactly the same but without the "quick and easy" (wrong) indication. So _In just a few moments, you will become part of a community consisting of more than 250,000 merchants. You are on the way to creating your own unique online store that you can manage easily every day._ localized in _Install_, thank you.
@colinegin, is it okay for you?

Reply:
Fixed